src/Umbraco.Web.UI/Umbraco/config/lang/da.xml

```diff
@@ -1824,12 +1824,12 @@ Mange hilsner fra Umbraco robotten
     <key alias="tooltipForPropertyActionsMenu">Åben egenskabshandlinger</key>
   </area>
   <area alias="blockEditor">
-    <key alias="headlineCreateBlock">Opret ny blok</key>
+    <key alias="headlineCreateBlock">Vælg Element-Type</key>
     <key alias="headlineAddSettingsElementType">Tilføj en indstillings afsnit</key>
     <key alias="headlineAddCustomView">Tilføj visning</key>
     <key alias="headlineAddCustomStylesheet">Tilføj stylesheet</key>
     <key alias="headlineAddThumbnail">Vælg billede</key>
-    <key alias="labelcreateNewElementType">Opret ny</key>
+    <key alias="labelcreateNewElementType">Opret ny Element-Type</key>

```

Comment:
@nielslyngsoe I think the correct term in Danish is `elementtype` like we have `dokumenttype`. It is what we in Danish call "sammensatte ord" 😁 
https://sproget.dk/raad-og-regler/typiske-problemer/et-eller-flere-ord

It would be `Elementtype` if it is in start of the sentence.

Reply:
I saw this approach begin used earlier, so I thought that was the right approach.

But now you made me look it up, and from my research, it seems that you are right to make it `elementtype`. both because we can merge names in danish but also because its not a "person-, virksomheds-, eller stednavn" so it should be with lowercase. So I learned something new today.

I think this needs to be corrected in general, so please make a PR for that, so we can have it consistent. Thanks

Reply:
Yeah, I think there are some general translations from English to Danish, where it can't be translated directly.

"Tilføj en indstillings elementtype" would probably be "Tilføj en indstillingselementtype" or "Tilføj en indstilling for elementtypen".